### PR TITLE
Update README with OCP UI instructions

### PR DESCRIPTION
**Pull request by mrizzi**
Rendered version https://github.com/mrizzi/xavier-integration/blob/update-README/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,17 @@ Instructions from https://access.redhat.com/documentation/en-us/red_hat_fuse/7.3
...
+### Using OCP Web UI
+1. click on `Create Project` button to create a new project
+1. use `migration-analytics` as value for the name
+1. once the project has been created and added to the list of the available project, click on `migration-analytics`
+1. from the `migration-analytics` Overview page, click on `Import YAML / JSON` button
+1. copy and paste the content of the [analytics_template.json](src/main/resources/okd/analytics_template.json) (using the `Raw` button will let you have the plain text version of this file)
+1. click on `Create` button
+1. check that `Process the template` is selected (no need to select `Save template`)
+1. click on `Continue` button
+1. click on `Create` button
```
Comment (carlosthe19916) [inline README.md]: @mrizzi I think you could add something like "(No need to change the form values unless you want to customize it)"

**Review by carlosthe19916 — COMMENTED** [review]
Everything looks great to me. I just suggest adding a simple clarification on
- "click on `Create` button"

File: README.md (hunk 1 above)
Comment (mrizzi) [reply README.md]: Thanks @carlosthe19916, i've added it

**Comment by carlosthe19916** [comment]
Thanks @mrizzi, I think this PR is ready to be merged

**Pull request merged**